.github/workflows/hacl-packages-create-branch.yml

```diff
@@ -1,11 +1,11 @@
-name: Update hacl-packages
+name: Create branch on hacl-packages
 
 on:
   pull_request:
     branches: [ main ]
```

Comment:
Should this get a `reopened` as well to create the branch again?

Reply:
I just tested on a fork and, indeed, reopening a PR does not trigger the worflow.